web/src/views/HomeView/StatsBar.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <v-container
+    class="grey darken-3 pa-0"
+    fluid
+  >
+    <v-row class="py-6">
+      <template v-for="stat in stats">
+        <v-col
+          :key="stat.name"
+          class="py-0 flex-grow-1"
+          md="2"
+          sm="4"
+          cols="6"
+        >
+          <SingleStat
+            :name="stat.name"
+            :value="stat.value.toString()"
+            :description="stat.description"
+          />
+        </v-col>
+        <!-- TODO dividers destroy the grid system breakpoints
+        <v-divider
+          :key="stat.name + '-divider'"
+          vertical
+          class="grey"
+        />
+        -->
+      </template>
+    </v-row>
+  </v-container>
+</template>
+
+<script>
+import filesize from 'filesize';
+import { mapState } from 'vuex';
+import SingleStat from '@/views/HomeView/SingleStat.vue';
+
+export default {
+  name: 'StatsBar',
+  components: { SingleStat },
+  computed: mapState({
```

Comment:
I think we agreed that this probably shouldn't even use Vuex, so it's not critical in the long-term, but just FYI, check out these two sections:
* https://vuex.vuejs.org/guide/modules.html#binding-helpers-with-namespace
* https://vuex.vuejs.org/guide/state.html#object-spread-operator

I think we should always use these patterns in other code. I don't think it's critical that we make a change here.